content/liveobjects/storage.textile

```diff
@@ -0,0 +1,62 @@
+
+You can store your objects outside of Ably by obtaining the channel objects via the "REST API":/docs/liveobjects/rest-api-usage#fetching-objects and storing them in your own database. This is useful if you want to keep a permanent record of the objects on a channel or if you want to perform analytics on the data.
+
+In order to receive notifications when the objects on a channel are updated, use "inband objects":/docs/liveobjects/inband-objects to receive updates as regular channel messages.
+
+<aside data-type='note'>
+<p>If you're interested in exporting objects to your own systems using "integration rules":/docs/integrations/webhooks, please "get in touch":/contact with your requirements.</p>
+</aside>
+
+h2(#operation-persistence). Operation storage
+
+When you update an object, the change is expressed as an "operation":/docs/liveobjects/concepts/operations that is sent as an "object message":/docs/metadata-stats/stats#messages on the channel. Like all messages, Ably stores object messages for 2 minutes by default.
+
+This means that if a client disconnects from Ably for a short period of time, it can automatically retrieve any operations it may have missed when it reconnects. If a client disconnects for longer than 2 minutes, the client will be sent the latest state of the objects on the channel (which are durably stored for 90 days) when it reconnects, ensuring the client remains fully synchronized.
+
+Operations themselves are not included in the "history":https://ably.com/docs/storage-history/history or "rewind":/docs/channels/options/rewind backlog of a channel. Instead, you should interact with objects directly via the client library.
+
+h2(#limits). Object count and size limits
+
+There is a maximum number of objects that can be stored on a "channel":/docs/pricing/limits#channel, which is configured to 100 objects by default.
+
+<aside data-type='note'>
+<p>If you're interested storing a larger number of objects on a channel, please "get in touch":/contact with your requirements.</p>
+</aside>
+
+A @LiveCounter@ is a double-precision floating-point number and has a size of 8 bytes.
+
+The size of a @LiveMap@ object is calculated as the sum of the length of all keys plus the size of all values where:	
+
+* @string@ values are the length of the string
+* @number@ values are 8 bytes
+* @boolean@ values are 1 byte
+* @bytes@ values are the length of the byte array
+
+The maximum allowed size of a single @LiveMap@ object is the same as the "message size limit":/docs/pricing/limits#message for your package. This is because objects are "synchronized":/docs/liveobjects/concepts/synchronization#client-objects to the client as channel messages.
+
+<aside data-type='note'>
+<p>Taken together, the implicit limit on the total size of your channel objects is the "message size limit":/docs/pricing/limits#message for your package multiplied by the maximum number of objects allowed on a channel (currently 100).</p>
```

Comment:
I'd just state this as `100` and drop the 'currently'.

Reply:
Fixed in: [409989b](https://github.com/ably/docs/pull/2591/commits/409989ba2fa2c08eae17dc52127ffc5ca9d101a6)